cmd/benchmark.go

```diff
@@ -1,267 +1,81 @@
-// 				results.Total = results.Total + 1
-// 			case r := <-resultsCh:
-// 				log.Infof("[benchmark] results chainID=%s duration=%dms err=%s", r.Transaction.ObjectId, r.Duration, r.Error)
-// 				results.Measured = results.Measured + 1
-//
-// 				if r.Error != "" {
-// 					results.Errors = append(results.Errors, r.Error)
-// 					results.Failures = results.Failures + 1
-// 				} else {
-// 					results.Durations = append(results.Durations, r.Duration)
-// 					results.Successes = results.Successes + 1
-// 				}
-// 			}
-// 		}
-//
-// 		sum := 0
-// 		for _, v := range results.Durations {
-// 			sum = sum + v
-// 		}
-//
-// 		if results.Successes > 0 {
-// 			results.AverageDuration = sum / results.Successes
-//
-// 			sorted := make([]int, len(results.Durations))
-// 			copy(sorted, results.Durations)
-// 			sort.Ints(sorted)
-//
-// 			results.MinDuration = sorted[0]
-// 			results.MaxDuration = sorted[len(sorted)-1]
-// 			p95Index := int64(math.Round(float64(len(sorted))*0.95)) - 1
-// 			results.P95Duration = sorted[p95Index]
-// 		}
-//
-// 		out, _ := json.MarshalIndent(results, "", "  ")
-// 		fmt.Println(string(out))
-// 	},
-// }
-//
-// func init() {
-// 	rootCmd.AddCommand(benchmark)
-// 	benchmark.Flags().IntVarP(&benchmarkConcurrency, "concurrency", "c", 1, "how many transactions to execute at once")
-// 	benchmark.Flags().IntVarP(&benchmarkDuration, "iterations", "i", 10, "how many seconds to run benchmark for")
-// 	benchmark.Flags().IntVarP(&benchmarkTimeout, "timeout", "t", 0, "seconds per transaction to wait before timing out")
-// 	benchmark.Flags().IntVarP(&benchmarkStartDelay, "start-delay", "d", 0, "how many seconds to wait before kicking off the benchmark; useful if network needs to stablize first")
-// 	benchmark.Flags().StringVarP(&benchmarkStrategy, "strategy", "s", "", "(tps|tps-no-ack) - tps will measure every transaction, tps-no-ack will only measure the last round of transactions")
-// 	benchmark.Flags().BoolVar(&enableJaegerTracing, "jaeger-tracing", false, "enable jaeger tracing")
-// 	benchmark.Flags().BoolVar(&enableElasticTracing, "elastic-tracing", false, "enable elastic tracing")
-// 	benchmark.Flags().StringVar(&tracingTagName, "tracing-name", "", "adds tag to tracing for lookup")
-// }
+	"github.com/quorumcontrol/tupelo-go-sdk/gossip/client"
+	"github.com/quorumcontrol/tupelo-go-sdk/gossip/client/pubsubinterfaces/pubsubwrapper"
+	"github.com/quorumcontrol/tupelo-go-sdk/p2p"
+	"github.com/quorumcontrol/tupelo/gossip/benchmark"
+	"github.com/quorumcontrol/tupelo/nodebuilder"
+	"github.com/spf13/cobra"
+	"gopkg.in/src-d/go-log.v1"
+)
+
+var benchmarkConcurrency int
+var benchmarkTimeout int
+var benchmarkDuration int
+var benchmarkStartDelay int
+
+// benchmark represents the shell command
+var benchmarkCmd = &cobra.Command{
+	Use:    "benchmark",
+	Short:  "runs a set of operations against a network at specified concurrency",
+	Hidden: true,
```

Comment:
What does this do? Do we want it?

Reply:
keeps it out of help (this was in the original benchmark)... not sure if we want it in or out.

Reply:
OK, makes sense. Yeah, I don't have a strong opinion either way.